### DISCUSSION TITLE
Borderless by default

### DISCUSSION THREAD

**Discussion by saturdayxiii** [RamonUnch/AltDrag#454] [Ideas]
My favorite feature about AltSnap has become the ability to make windows borderless.  Maybe going through AltSnap isn't the best way to accomplish this, but is there a way to have program windows automatically launch in a borderless mode? 

I've noticed Media Player Classic actually has this feature built in its preferences, then if you hover your mouse by the the top right corner of the window the min-max-ex buttons appear.  I think I would love it if this were the standard way for windows to work.

**Comment by Ichisich** [comment]
AltSnap is non-intrusive, has no automatic process detection, and only activates when you press Alt (or your set hotkey).

Additionally, borderless mode can break many applications (try to make any browser borderless) and thus should preferably set by the application itself, e.g. MPC.
If you want a more intrusive program that can autodetect application windows and set them borderless automatically at start, there is 
[https://github.com/Codeusa/Borderless-Gaming](https://github.com/Codeusa/Borderless-Gaming)

You can set up a favorite list of programs you want borderless/resized/fullscreen/alwaysOnTop etc. when launched.
You won't get the min-max-ex buttons to appear, but who needs them, when you have AltSnap.
